plugins/connection/libvirt_qemu.py

```diff
@@ -11,8 +11,8 @@
 DOCUMENTATION = """
 ---
 author:
-    - Jesse Pretorius <jesse@odyssey4.me>
-connection: community.libvirt.libvirt_qemu
+    - Jesse Pretorius (@odyssey4me)
```

Comment:
BTW, if you want to keep the email address, you can do:
```suggestion
    - Jesse Pretorius (@odyssey4me) <jesse@odyssey4.me>
```

Reply:
Great, thanks @felixfontein! I have updated this to include the email.